File: tutorial01/java-gradle/src/main/java/org/luwrain/app/example/MainLayout.java
package org.luwrain.app.example;

import org.luwrain.core.*;
import org.luwrain.app.base.*;
import org.luwrain.controls.*;

public class MainLayout extends LayoutBase {
private NavigationArea area;
public MainLayout(App app) {
super(app);
this.area = new SimpleArea(getControlContext(), "hello, world", new String[] {"a","b"});
setAreaLayout(this.area, null);
}
}

File: tutorial01/kotlin/src/main/kotlin/org/luwrain/app/tutorial01/MainLayout.kt

package org.luwrain.app.tutorial01

import org.luwrain.core.*
import org.luwrain.core.events.*
import org.luwrain.controls.*
import org.luwrain.app.base.*

class MainLayout: LayoutBase {
    val area : NavigationArea

    constructor(app: App): super(app)  {
	this.area = SimpleArea(getControlContext(), "Hello, world", arrayOf("Hello, world!", ""))
	setAreaLayout(area, null);
    }
}
